### ISSUE TITLE
How to choose proper hosting size?

### ISSUE THREAD

**Issue by seaner90** [knadh/listmonk#816]
Hi,

I'm going to host listmonk on aws lightsail, but not sure which one is good for hosting listmonk, is there any minimum RAM requirement?

I have one list with about 5000 subscribers(all double opted-in), send 5~10 campaigns a month, does **512 MB Memory with swap** enough for this? or maybe **1 GB Memory & 1 Core Processor**

Thank you

**Comment by knadh** [comment]
Yep, way more than enough. Check out the `Performance` section here: https://listmonk.app

**Comment by seaner90** [comment]
@knadh Thank you for quick reply, however, I'm going to host listmonk and postgresql on the same lightsail instance, your website only explain listmonk performance, but didn't tell your application psql consumption, is it ok if I host them on 512mb lightsail? :)

**Comment by knadh** [comment]
A few thousand subscribers, I think it'll run fine.

**Comment by seaner90** [comment]
thank you very much !
